src/scss/_shabad-page.scss

```diff
@@ -73,6 +77,10 @@ blockquote {
   flex-grow: 1;
 }
 
+.mixed-view-baani-transliteration-shahmukhi {
+  float: right;
+}
+
```

Comment:
@saintsoldierx can we try using flex-box properties here. Just for consistency and also float needs to be cleared as well,  otherwise generally they end up causing layout issues

Reply:
![image](https://user-images.githubusercontent.com/27791330/93805489-ae21c300-fbfc-11ea-8118-5ada667c2399.png)
 Would this be sufficient @gurjit03 ?